### DISCUSSION TITLE
Extend `addValidationError` (or similar) to separate positive and negative feedback

### DISCUSSION THREAD

**Discussion by noor-codes** [keystonejs/keystone#7279] [Feature requests]
### Background 
I am am definitely a big fan of `addValidationError` function in `validateInput` hook which takes error as a string and shows the error on the admin UI. 

### The Problem

- When we kick an error, we usually aware the user what things went wrong but that's 50% of the story. 
- Right now we can tell the user what went wrong by `addValidationError` function. 
- We also need to tell the user how to solve the error so they can move on and get more things done. 

- The more tasks a user can perform in a system, the more likely they will love the platform. 

### Possible Solution
- If I had another function like `addInstructionMessage` function with the same behavior like `addValidationError`  where I could instructions user have to get the job done with a blue background and a separate box. It would separate the negative and positive messages from each other. A much better User Experience.

- Right now I have to put the positive and negative messages in the same place which logically doesn't make sense to me. 

Let me know what you think! 
Thanks.

![Screen Capture_select-area_20220204150840](https://user-images.githubusercontent.com/28804764/152517170-b09f6a85-7e78-4c5c-9438-360ee83b58f7.png)

**Comment by dcousens** [comment]
Thanks @noor-codes for the feedback!
We'll feed this into the design team pipeline and see what this could feature could look like or if there are any other edge cases.

**Reply by noor-codes** [reply]
Sounds Great!
